syntax = "proto3";

service Actions {
    rpc ScheduleLike (ActionsRequest) returns (ActionsReply) {}
    rpc UnScheduleLike (ActionsRequest) returns (ActionsReply) {}
    rpc HideProfile (ActionsRequest) returns (ActionsReply) {}
}

message ActionsRequest {
    int32 user_id = 1;
}

message ActionsReply {
    bool success = 1;
    string message = 2;
    optional int64 s_number = 3;
}
